src/V3EmitCBase.cpp

```diff
@@ -0,0 +1,213 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
```

Comment:
I appreciate it's extra work, but could you push making V3EmitCBase.cpp (with just the same functions as before) as a separate commit to master?

Reply:
No problem at all. I have separated out the pure refactor changes and pushed them to master separately.